### DISCUSSION TITLE
How to specify and represent a point on the x-axis

### DISCUSSION THREAD

**Discussion by junghoonshin3** [patrykandpatrick/vico#423] [Questions]
hello. I'm trying to draw a line chart that shows time on the x-axis and exchange rate information on the y-axis.
I would like to express the x-axis at 9, 10, 11, 12, 13, 14, and 15 o'clock.
For example, if there are 10 exchange rate data corresponding to 9 o'clock,
I want to display only 9 o'clock on the x-axis. Is there a way?

**Comment by patrickmichalik** [comment]
Hello! Yes, this is possible. Please use `getXStep` to set the _x_ step to the _x_ delta corresponding to 60 minutes. Depending on the specifics of the desired result, `AxisItemPlacer.Horizontal` may also come in handy.
